src/docs/demos/modal/modal-demo.controller.js

```diff
@@ -1,13 +1,9 @@
 class controller {
-  constructor ($ngbsModal) {
-    this.$ngbsModal = $ngbsModal;
+  constructor($log, $ngbsModal) {
+    Object.assign(this, { $log, $ngbsModal });
```

Comment:
I always sort of wish ngInject did this automatically.  Are you using ngInject or maybe not uglifying the JavaScript?